hub/management/commands/import_wildlife_trusts_regions.py

```diff
@@ -0,0 +1,45 @@
+from django.conf import settings
+
+import pandas as pd
+
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand
+
+
+class Command(BaseImportFromDataFrameCommand):
+    help = "Import Wildlife trust reserves"
+
+    data_file = (
+        settings.BASE_DIR / "data" / "wildlife_trust_regions_in_constituencies.csv"
+    )
+    cons_row = "PCON19NM"
+    uses_gss = False
+    message = "Importing constituency Wildlife Trust regions data"
+
+    data_sets = {
+        "wildlife_trust_regions": {
+            "defaults": {
+                "label": "Wildlife Trust Regions",
+                "description": "The Wildlife Trust Region(s) that a constituency falls under.",
+                "data_type": "text",
```

Comment:
This is causing a problem as it means a constituency can only be in one trust which ends up being the last one with that constituency in the sheet. It might make more sense for this to work like groups, although then not filterable although we should probably create a ticket to allow filtering on group names as you can do that in postgres I think.